### DISCUSSION TITLE
🙏 Help: Designer missing buttons for download theme?

### DISCUSSION THREAD

**Discussion by nmkeik** [public-ui/kolibri#6802] [Q&A] [answered]
### Summary

Ich habe gerade angefangen mich aus persönlichem Interesse in KoliBri einzulesen und ein wenig auszutesten. Ich wollte im Designer ein eigenes Theem erstellen, wiederum fehlen bei mir die Knöpfe für THEME ERSTELLEN und THEME HERUNTERLADEN, genauso wie die Schaltfläche Theme laden. Der einzige Vorhandene Knopf unten ist ALLE ÄNDERUNGEN VERWERFEN. (Habe sschon nachgeschaut ob es an meinem Browser oder Rechner liegt, aber bei allen Browser von verschiedenen Geräten mit auch verschiedenen Betriebssystemen habe ich überall das gleiche Problem.)

Da ich noch komplett neu zu KoliBri bin, bin ich mir unsicher ob dies ein Bug ist oder ich irgendwas übersehen habe, weshalb ich hier im Q&A nachfragen wollte.

Mir war während dem lesen der Dokumentation aufgefallen, dass ein paar links zu nicht mehr vorhanden Seiten führen. Da ich wie gesagt sehr neu hier bin war ich mir unsicher ob bzw. wo man das melden soll?

Falls mir jemand weiter helfen kann würde mich das sehr freuen.

Viele Grüße : )

![Screenshot_20240909_224948](https://github.com/user-attachments/assets/99377b19-4cc3-4f0a-8639-cdd0fef8d04e)

**Comment by deleonio** [comment, marked as answer]
Hallo @nmkeik,

vielen Dank für Ihr Interesse und Ihr Feedback, was uns sehr hilft.

Der Designer wurde mit der Version 1 von KoliBri veröffentlicht und bot so eine Möglichkeit, interaktiv ein eigenes Theme zu erstellen. In der Praxis hat sich dann aber mehr und mehr gezeigt, dass die Teams, die diese Themes umsetzen und pflegen, das viel lieber in einer IDE tun.

Aus dieser Erkenntnis haben wir dem Theme-Repository-Ansatz mehr Aufmerksamkeit geschenkt und dieses vollständig mit optischen Regressionstests umgesetzt und als Template-Repo veröffentlicht.

Die Features des Designers wurden mehr reduziert.

---

Ich würde Ihnen empfehlen, sich einmal folgendes Repository exemplarisch anzuschauen: https://github.com/public-ui/kolibri-theme-bwst

Wenn Sie sich das Clonen, Installieren und Starten, können Sie interaktiv Style-Änderungen in den SCSS-Files vornehmen und live im Browser die Änderungen verfolgen. Auf diesem Weg werden heute Themes erstellt, getestet und weiterentwickelt.
